### ISSUE TITLE
Make ComputeTimeStampJob runnable on the cluster

### ISSUE THREAD

**Issue by Marvin84** [rwth-i6/i6_core#532]
According to #523 this job is supposesd to run as a`mini_task`, shown [here](https://github.com/rwth-i6/i6_core/blob/main/mm/alignment.py#L576). However, I had an issue when running this job on the `LibriSpeech` 960h. My local machine had serious memory problems crashing after a few minutes. Even on the cluster, I had to request more memory for a successful run.
I suggest we make this job runnable in the cluster by default, and add settable requirements for it.

**Comment by michelwi** [comment]
> I suggest we make this job runnable in the cluster [..] and add settable requirements for it.

I agree. a recurring trope how this is done would be 
```
# in __init__
self.rqmt = None

# in tasks
yield Task("run", rqmt=self.rqmt, mini_task=self.rqmt is not None)
```
in this way the job is by default run in the LocalEngine but anyone can make it a regular task by setting an explicit rqmt, e.g.
```
my_job = ComputeTimeStampJob(*args)
my_job.rqmt = {"time": 1.5, "mem": 4.5}
```

> in the cluster by default

not sure if I would change the default behavior.. if it always runs into these problems maybe. but until now we seem to have used the job successfully it seems.
What was the memory required and were you doing anything special?

**Comment by Marvin84** [comment]
Thanks @michelwi , your proposed solution is certainly the way to go.

>  What was the memory required and were you doing anything special?

I followed @christophmluscher suggestion, due to his previous experience with this job and LBS 960h, and set `mem` to 6, I did not try other values and only know that `mem` equal to 1 would fail.

**Comment by SimBe195** [comment]
Fixed by #533.
